addons/analytic/models/analytic_plan.py

```diff
@@ -282,6 +282,7 @@ def _sync_plan_column(self):
                     'relation': 'account.analytic.account',
                     'store': True,
                     'on_delete': 'restrict',
+                    'index': True,
```

Comment:
```suggestion
```
Doing this will render the following manual creation of the index useless:
https://github.com/odoo/odoo/blob/f922f09b83c68dff36c064d20709a6e6ba4541dc/odoo/tools/sql.py#L510-L513

But we want to keep a "NOT NULL" index (which can't be specified since `index` is just a Boolean on `ir.model.fields`).

I would recommend setting `index` to `True` on the field after the call to `create_index()`.

That way, we signal the ORM that the field is indexed, removing the logs, while keeping the index memory in check :nerd_face: